index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Def Con summit's home page</title>
+    <link rel="stylesheet" href="static/css/main.css">
+    <link rel="stylesheet" href="https://use.typekit.net/tlw1tez.css">
+    <link rel="stylesheet" href="https://use.typekit.net/tlw1tez.css">
+    <link rel="stylesheet" href="https://use.typekit.net/qlg0xnl.css">
+</head>
+
+<body>
+    <header>
+        <nav class="nav_social">
+            <ul>
+                <li><a href="about.html"><img src="static/images/Facebook.png" alt="Facebook"></a></li>
+                <li><a href=""><img src="static/images/twitter.png" alt="Twitter"></a></li>
+                <li><a href="">English</a></li>
+                <li><a href="">My Page</a></li>
+                <li><a href="">Logout</a></li>
+            </ul>
+        </nav>
+        <div class="nav_bar">
+            <div class="burger">
+                <img src="static/images/burger.png" alt="Burger menu button">
+            </div>
+            <div class="logo">
+                <a href="index.html"><img src="static/images/defcon_logo.png" alt="CC global summit logo"></a>
+            </div>
+            <nav class="menu_wide">
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a class="campaign" href="">CC Campaign</a></li>
+                </ul>
+            </nav>
+        </div>
+
+
+    </header>
+
+    <dialog class="modal_menu">
+        <div class="close">&times;</div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+            </ul>
+        </nav>
```

Comment:
- **[OPTIONAL]** Kindly use a transparent background for the mobile menu for a better UI 👇🏽 

> There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).